apps/condo/pages/employee/[id]/index.tsx

```diff
@@ -279,42 +305,65 @@ export const EmployeePageContent = ({
 }
 
 export const EmployeeInfoPage: PageComponentType = () => {
-    const { query } = useRouter()
+    const { query, push } = useRouter()
     const { link } = useOrganization()
     const intl = useIntl()
-    const UpdateEmployeeMessage = intl.formatMessage({ id: 'employee.UpdateTitle' })
+    const LoadingInProgressMessage = intl.formatMessage({ id: 'LoadingInProgress' })
     const ErrorMessage = intl.formatMessage({ id: 'errors.LoadingError' })
+    const DeleteUserMessage = intl.formatMessage({ id: 'employee.Notification.deleteUser' })
 
     const employeeId = String(get(query, 'id', ''))
+
+    const isFirstRender = useRef<boolean>(true)
+
     const { obj: employee, loading, error, refetch } = OrganizationEmployee.useObject(
         {
             where: {
                 id: employeeId,
             },
         }
     )
+
+    useEffect(() => {
+        if (isFirstRender.current) {
+            isFirstRender.current = false
+            return
+        }
+        if (!loading && !employee) {
+            push('/employee/')
+        }
+    }, [employee, loading, push])
```

Comment:
What is it?

Reply:
It is necessary that we do not have pages like these
<img width="1512" alt="Screenshot 2024-12-12 at 13 25 55" src="https://github.com/user-attachments/assets/b9d8ec08-eb88-4a13-a015-2fef9fc94ff9" />
This employee was deleted

Reply:
this is something strange, better tell the user that the employee was not found

I haven't seen similar logic with redirect